gateway/pom.xml

```diff
@@ -125,6 +125,14 @@
       <artifactId>json</artifactId>
       <version>20230618</version>
     </dependency>
+    <!-- testcontainers dependency added to run SendMessageRabbitmqIT
+    with rabbitmq and smtp containers.-->
```

Comment:
maybe the comment is not necessary, I think we can assume that if the dependency is added (especially with a test scope), it's for testing purposes ;-) and the reference to smtp is a bit confusing for this dependency in particular